apiserver/facades/agent/uniter/export_test.go

```diff
@@ -47,6 +47,7 @@ func PatchGetStorageStateError(patcher patcher, err error) {
 	patcher.PatchValue(&getStorageState, func(st *state.State) (storageAccess, error) { return nil, err })
 }
 
-func (n *NetworkInfoBase) MachineNetworkInfos(spaceIDs ...string) (map[string]params.NetworkInfoResult, error) {
-	return n.machineNetworkInfos(spaceIDs...)
+func (n *NetworkInfoIAAS) MachineNetworkInfos() (map[string]params.NetworkInfoResult, error) {
```

Comment:
I hate doing this, but an import loop prevents package-local testing that requires `JujuConnSuite`.